controller_manager:
  ros__parameters:
    update_rate: 50

diff_drive_controller:
  ros__parameters:
    type: diff_drive_controller/DiffDriveController
    left_wheel: left_wheel_joint
    right_wheel: right_wheel_joint
    wheel_separation: 0.5
    wheel_radius: 0.1
    publish_odometry: true
    odom_topic: /odom
    base_frame_id: base_footprint
    odom_frame_id: odom
    twist_topic: /cmd_vel
